egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -368,11 +375,16 @@ def CreateReverberatedCopy(input_dir,
                            max_noises_per_minute  # maximum number of point-source noises that can be added to a recording according to its duration
                            ):
     
+    wav_scp = ParseFileToDict(input_dir + "/wav.scp", value_processor = lambda x: " ".join(x))
     if not os.path.isfile(input_dir + "/reco2dur"):
         print("Getting the duration of the recordings...");
-        data_lib.RunKaldiCommand("wav-to-duration --read-entire-file=true scp:{0}/wav.scp ark,t:{0}/reco2dur".format(input_dir))
+        read_entire_file="false"
+        for value in wav_scp.values():
+            if "sox" in value and "speed" in value:
```

Comment:
Add a comment here saying that we might want to check for other sox commands which modify the header as we come across these cases in our data.